inputfiles/overridingTypes.jsonc

```diff
@@ -1,3411 +1,3487 @@
-                            "signature": {
-                                "0": {
-                                    "param": [
-                                        {
-                                            "name": "transfer",
-                                            "overrideType": "Transferable[]"
-                                        }
-                                    ]
-                                }
-                            }
-                        }
-                    }
-                },
-                "events": {
-                    "event": [
-                        {
-                            "name": "message",
-                            "type": "MessageEvent"
-                        },
-                        {
-                            "name": "messageerror",
-                            "type": "MessageEvent"
-                        }
-                    ]
+              }
+            }
+          }
+        }
+      },
+      "SVGSymbolElement": {
+        "extends": "SVGElement"
+      },
+      "FormData": {
+        "methods": {
+          "method": {
+            "append": {
+              "name": "append",
+              "flavor": "Web",
+              "overrideSignatures": [
+                "append(name: string, value: string | Blob, fileName?: string): void"
+              ]
+            },
+            "set": {
+              "name": "set",
+              "flavor": "Web",
+              "overrideSignatures": [
+                "set(name: string, value: string | Blob, fileName?: string): void"
+              ]
+            }
+          }
+        }
+      },
+      "HTMLSelectElement": {
+        "properties": {
+          "property": {
+            "selectedOptions": {
+              "name": "selectedOptions",
+              "overrideType": "HTMLCollectionOf<HTMLOptionElement>"
+            },
+            "labels": {
+              "name": "labels",
+              "overrideType": "NodeListOf<HTMLLabelElement>"
+            }
+          }
+        },
+        "overrideIndexSignatures": [
+          "[name: number]: HTMLOptionElement | HTMLOptGroupElement"
+        ]
+      },
+      "HTMLDataListElement": {
+        "properties": {
+          "property": {
+            "options": {
+              "name": "options",
+              "overrideType": "HTMLCollectionOf<HTMLOptionElement>"
+            }
+          }
+        }
+      },
+      "HTMLDirectoryElement": {
+        "element": [
+          {
+            "name": "dir"
+          }
+        ]
+      },
+      "HTMLFontElement": {
+        "element": [
+          {
+            "name": "font"
+          }
+        ]
+      },
+      "FontFace": {
+        "properties": {
+          "property": {
+            "display": {
+              "name": "display",
+              "overrideType": "FontDisplay"
+            }
+          }
+        }
+      },
+      "FontFaceDescriptors": {
```

Comment:
where's the `dictionaries`? I think I need more information from the contribute guide, like what's the meaning of those and where I can learn about and how to modify `dictionary` type etc.

Reply:
could you please improve the contribute guide about these information?

Reply:
This one is more about IDL rather than the JSON itself. You need to know [look at the spec](https://w3c.github.io/csswg-drafts/css-font-loading/#fontface-interface) from the MDN "Specifications" section to see it's `interface FontFaceDescriptors` or `dictionary FontFaceDescriptors`.